### ISSUE TITLE
General Problem with creating a Kanban 

### ISSUE THREAD

**Issue by mpflugi** [arakkkkk/kanban.nvim#31]
Sorry for creating an issue. I installed your plugin. I will receive a problem with the gsub function while trying to create a kanban:

Maby you could help me out.
writer lua 4 - bad argument #3 to 'gsub' (string/function/table expected

 1. markdown/writer.lua4 in function write
 2. kanban.lua 87 in function ........ kanban.lua 72

Pretty new to neovim - I followed the installation guide

many thanks in advance

**Comment by arakkkkk** [comment]
@mpflugi 
Thank you for creating an issue.

I can't reproduce your error. It may be due to the version of neovim you're using.

Could you please tell me the version of your neovim environment?

**Comment by mpflugi** [comment]
Hi,

thank you for your response - I was able to fix most of it. I guess it is
because I am using windows. So I changed directories etc. Right now
everything works so far. The only thing I could not manage so far is the
possibility to save the description in an other File. But I don't need this
functionality so I disabled the possibility in the keymap.

best regards
Markus

Am Do., 21. März 2024 um 11:44 Uhr schrieb arakkk ***@***.***
